templates/zerver/help/configure_email_notifications.md

```diff
@@ -0,0 +1,15 @@
+# Configure email notifications
+
+Zulip allows you to configure your email notification preferences. You have a choice to receive or not receive email notification when you're offline.
```

Comment:
it should be **to recieve or not recieve email notifications** instead of **to recieve or not recieve email notification**